setup.py

```diff
@@ -64,9 +64,10 @@ def package_vars(version_file):
     keywords="skeleton",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
+    package_data={"example": ["data/*.txt"]},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt"],
+    install_requires=["docopt", "setuptools"],
```

Comment:
I've added `setuptools` as an explicit requirement since the script uses `pkg_resources`.  It is safe to assume `setuptools` is installed at install-time, but we want to prevent someone from yanking it out from under us later.

Reply:
> If one of the other devs can get the tests to run locally then I will withdraw my review.

This worked for me in a clean env, but I ran this after @felddy added 91ef4dd, so I don't know if that made the difference or not.
```
(skeleton-python-library) daver@blinky ~/code/cisagov/skeleton-python-library ❱ pytest
========================================================================= test session starts =========================================================================
platform darwin -- Python 3.7.2, pytest-4.6.2, py-1.8.0, pluggy-0.12.0 -- /Users/daver/.pyenv/versions/3.7.2/envs/skeleton-python-library/bin/python3.7
cachedir: .pytest_cache
rootdir: /Users/daver/code/cisagov/skeleton-python-library, inifile: pytest.ini
plugins: cov-2.7.1
collected 14 items                                                                                                                                                    

tests/test_example.py::test_version PASSED                                                                                                                      [  7%]
tests/test_example.py::test_log_levels[debug] PASSED                                                                                                            [ 14%]
tests/test_example.py::test_log_levels[info] PASSED                                                                                                             [ 21%]
tests/test_example.py::test_log_levels[warning] PASSED                                                                                                          [ 28%]
tests/test_example.py::test_log_levels[error] PASSED                                                                                                            [ 35%]
tests/test_example.py::test_log_levels[critical] PASSED                                                                                                         [ 42%]
tests/test_example.py::test_log_levels[critical2] XFAIL                                                                                                         [ 50%]
tests/test_example.py::test_division[1-1-1] PASSED                                                                                                              [ 57%]
tests/test_example.py::test_division[2-2-1] PASSED                                                                                                              [ 64%]
tests/test_example.py::test_division[0-1-0] PASSED                                                                                                              [ 71%]
tests/test_example.py::test_division[8-2-4] PASSED                                                                                                              [ 78%]
tests/test_example.py::test_division[0-0-0] XFAIL                                                                                                               [ 85%]
tests/test_example.py::test_slow_division SKIPPED                                                                                                               [ 92%]
tests/test_example.py::test_zero_division PASSED                                                                                                                [100%]

========================================================================== warnings summary ===========================================================================
/Users/daver/.pyenv/versions/3.7.2/envs/skeleton-python-library/lib/python3.7/site-packages/_pytest/mark/structures.py:337
  /Users/daver/.pyenv/versions/3.7.2/envs/skeleton-python-library/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html

---------- coverage: platform darwin, python 3.7.2-final-0 -----------
Name                      Stmts   Miss Branch BrPart  Cover   Missing
---------------------------------------------------------------------
src/example/__init__.py       3      0      0      0   100%
src/example/_version.py       1      0      0      0   100%
src/example/example.py       30      0      0      0   100%
---------------------------------------------------------------------
TOTAL                        34      0      0      0   100%

======================================================================= short test summary info =======================================================================
SKIPPED [1] tests/test_example.py:63: need --runslow option to run
XFAIL tests/test_example.py::test_log_levels[critical2]
XFAIL tests/test_example.py::test_division[0-0-0]
===================================================== 11 passed, 1 skipped, 2 xfailed, 1 warnings in 0.31 seconds =====================================================```

Reply:
That shouldn't affect anything. Could this be a python 3.7..3 thing?   @jsf9k is bleeding edge.

Reply:
`setuptools` was definitely installed locally when I ran, so it should make no difference.  I can try with python 3.7.2.

Reply:
No dice with 3.7.2.

Reply:
I feel like we should leave no man behind.  All for one and one for all and such.

Reply:
What was the last version that did pass tests for you?   Is it time to whip out `git bisect`? 
https://git-scm.com/docs/git-bisect

Reply:
I figured out the issue on my end.  I had an old `add.egg-info` directory lying around that was thwarting `pip`.  The proverbial "bad egg".

Thanks to @felddy for suggesting I look for stale files that might be obscured by `.gitignore`.